scipy/stats/tests/test_distributions.py

```diff
@@ -3236,6 +3236,48 @@ def test_logsf(self):
         y = stats.rayleigh.logsf(50)
         assert_allclose(y, -1250)
 
+    @pytest.mark.parametrize("floc", [-1, 0, 1, 2, 3])
+    @pytest.mark.parametrize("fscale", [.5, 1, 2, 3, 4])
```

Comment:
This might be a few too many. It would probably be enough to just use one set:
```
np.random.seed(0)  # or seed(0) if it's imported already
floc, fscale = np.random.rand(2)
```
But don't change anything unless someone else says something.